elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base), // needs to be #f0f0f0,
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
+  BorderRadius: pfe-var(ui--border-radius),
+  BorderWidth: pfe-var(ui--border-width),
+  Color: pfe-var(text),
+  FontSize: pfe-var(font-size),
+  Height: 36px,
+  Padding: 8px 6px,
+);
+
+:host {
+  display: inline-block;
+  position: relative;
+}
+
+::slotted(input) {
+  all: unset;
+  opacity: 0;
+  padding: pfe-local(Padding);
+  border-radius: pfe-local(BorderRadius);
+  border: pfe-local(BorderWidth) solid transparent;
+  height: pfe-local(Height) !important;
+}
+
+span input {
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  width: calc(100% - 14px);
```

Comment:
14px looks like a [magic number](https://css-tricks.com/magic-numbers-in-css/), what is that based on?

Reply:
It comes from adding the left and right padding and the width of the border.

padding-left: 6px + padding-right: 6px + border-left: 1px + border-right: 1px = 14px

Reply:
Using `box-sizing: border-box` on the input allowed me to remove the 14px from the width